### PR TITLE
chore: example incremental loader

### PR DESCRIPTION
**Pull request by thesayyn**
This is way faster and more incremental than anything out there.

### CODE REVIEW

**Comment by thesayyn** [comment]
https://github.com/bazel-contrib/rules_oci/issues/454

File: oci/private/tarball.bzl [hunk 1]
```diff
@@ -51,10 +51,9 @@ attrs = {
```
Comment (alexeagle) [inline oci/private/tarball.bzl:51]: update the doc on the `loader` attribute

File: oci/private/tarball.bzl [hunk 2]
```diff
@@ -51,10 +51,9 @@ attrs = {
 
             See the _run_template attribute for the script that calls this loader tool.
             """,
-        allow_single_file = True,
         mandatory = False,
```
Comment (alexeagle) [inline oci/private/tarball.bzl:54]: allow_single_file might have been load-bearing if someone just pointed this at an executable .sh file?

**Review by alexeagle — APPROVED** [review]

File: oci/private/tarball.bzl (hunk 2 above)
Comment (thesayyn) [reply oci/private/tarball.bzl:54]: hmm, i think i am gonna revert this change.

Comment (thesayyn) [reply oci/private/tarball.bzl:54]: can be done later if needed.

Comment (thesayyn) [reply oci/private/tarball.bzl]: reverted.

**Pull request merged**